shared/common-adapters/dropdown.tsx

```diff
@@ -119,15 +119,15 @@ type InlineDropdownProps = {
   containerStyle?: Styles.StylesCrossPlatform
   label: React.ReactNode
   onPress: () => void
-  type: 'Body' | 'BodySmall' | 'BodySmallSemibold'
+  type?: 'Body' | 'BodySmall' | 'BodySmallSemibold'
```

Comment:
hmm this seems possibly liable to make it easier for people to misuse this. WDYT of
```
type Props = { ... most of them ... } & 
({label: string; type: 'Body' | etc} | {label: React.ReactElement; type?: 'Body' | etc})
```

Reply:
Good call!